src/runtime/runtime_rpi3.go

```diff
@@ -0,0 +1,89 @@
+// +build rpi3
+
+package runtime
+
+import (
+	"machine"
+	"unsafe"
+)
+
+const tickMicros = int64(1)
+
+type timeUnit int64
+
+//go:export sleepticks sleepticks
+func sleepTicks(n timeUnit) {
+
+	machine.WaitMuSec(uint64(n))
+}
+
+func ticks() timeUnit {
+	return timeUnit(machine.SystemTime())
+}
+
+//go:export main
+func main() {
+	run()
+	machine.Abort()
+}
+
+func putchar(c byte) {
+	machine.MiniUART.WriteByte(c)
+}
+
+
+// abort is called by panic().
+func abort() {
+	machine.Abort()
+}
+
+
+// called once the memory allocator is ready and heap initialized
+func postinit() {
+	// Initialize .bss: zero-initialized global variables.
+	ptr := unsafe.Pointer(&_sbss)
+	for ptr != unsafe.Pointer(&_ebss) {
+		*(*uint32)(ptr) = 0
+		ptr = unsafe.Pointer(uintptr(ptr) + 4)
+	}
+
+	machine.MiniUART.Configure(machine.UARTConfig{})
+}
+
+// EnableInterruptController takes a set of on bits to turn on. It is usually
+// called with a single bit, like 1<<29 for AUX IRQ or 1<<1 for Timer 1 IRQ.
+// There are constants defined for most some interrupts in
+// src/device/rpi3.go.
+//func EnableInterruptController(bits uint32) {
+//	volatile.StoreUint32((*uint32)(dev.ENABLE_IRQS_1), bits)
+//}
+
+// EnableIRQ turns on the I in daif reg
+//func EnableIRQ() {
+//	dev.Asm("msr daifclr, #2")
+//}
```

Comment:
Such dead code shouldn't be kept around, to keep the diff clean. Feel free to put it in a separate branch for when you get interrupts to work, but don't leave around unused code.

Reply:
Yep, this code is in the upcoming PR that has interrupts working for the UART.  I deleted for this PR, however.